CyLR/src/CollectionPaths.cs

```diff
@@ -28,11 +29,81 @@ public static List<string> GetPaths(Arguments arguments, List<string> additional
                 {
                     "/root/.bash_history",
-                    "/var/logs"
+                    "/var/log",
+                    "/private/var/log/",
+                    "/.fseventsd",
+                    "/etc/hosts.allow",
+                    "/etc/hosts.deny",
+                    "/etc/hosts",
+                    "/System/Library/StartupItems",
+                    "/System/Library/LaunchAgents",
+                    "/System/Library/LaunchDaemons",
+                    "/Library/LaunchAgents",
+                    "/Library/LaunchDaemons",
+                    "/Library/StartupItems",
+                    "/etc/passwd",
+                    "/etc/group",
+                    ""
+                };
+
+                var proc_plist = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = "/usr/bin/find",
+                        Arguments = "/ -name \"*.plist\" -print",
+                        UseShellExecute = false,
+                        RedirectStandardOutput = true,
+                        CreateNoWindow = true
+                    }
+                };
+                proc_plist.Start();
+                while (!proc_plist.StandardOutput.EndOfStream)
+                {
+                    string line = proc_plist.StandardOutput.ReadLine();
+                    defaultPaths.Add(line);
+                };
+
+                var proc_bashshellhistory = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = "/usr/bin/find",
+                        Arguments = "/ -name \".bash_history\" -print",
+                        UseShellExecute = false,
+                        RedirectStandardOutput = true,
+                        CreateNoWindow = true
+                    }
+                };
+                proc_bashshellhistory.Start();
+                while (!proc_bashshellhistory.StandardOutput.EndOfStream)
+                {
+                    string line = proc_bashshellhistory.StandardOutput.ReadLine();
+                    defaultPaths.Add(line);
+                };
+
+
+
+                var proc_shellhistory = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = "/usr/bin/find",
+                        Arguments = "/ -name \".sh_history\" -print",
+                        UseShellExecute = false,
+                        RedirectStandardOutput = true,
+                        CreateNoWindow = true
+                    }
+                };
+                proc_shellhistory.Start();
+                while (!proc_shellhistory.StandardOutput.EndOfStream)
+                {
+                    string line = proc_shellhistory.StandardOutput.ReadLine();
+                    defaultPaths.Add(line);
```

Comment:
It might be good to pull these out into functions. Since the `find` commands only vary by argument, we could just pass that into a function and then that'll collapse this nicely.